department-of-veterans-affairs/vets-website
# MHV-55326 Update MR landing page and side nav for vitals

## Summary

- Added all remaining domains to the SideNav behind their respective feature flags.
- Added vitals content to the MR Landing Page, behind a feature flag.

## Related issue(s)

### [MHV-55326](https://jira.devops.va.gov/browse/MHV-55326) - Update Content on landing page and Left Navigation for Vitals ###

> Update Content on landing page and left navigation for Vitals-see latest designs
> 
> User Story: As a Medical Records User, I want to see updated content and left navigation so that the latest designs are displayed.
> 
> 
> 
> 
> Any content on the MR landing page related to vitals will need to be hidden behind the feature flag until Vitals is turned on.
> 
> Feature Flag Y/N ? Y
> 
> DataDog/Analytics Y/N ?
> 
> Manual Testing Y/N ? Y
> 
> Automated Testing Y/N ? N
> 
> Accessibility Testing Y/N ? N
> 
> UCD Validation Y/N Y
> 
> 
> 
> 
> Acceptance Criteria:
> 
> AC1  Left Navigation has been updated to latest design
> 
> AC2 Content has been updated to latest design landing page
> 
> AC3 Verify with UCD 
> AC4
> AC5
> 
> 
> 
> 
> Links to UCD:
> 
> Figma Link:
> 
> User Flow:
> 
> Mobile:
> 
> Desktop:
> 
> Other Design Notes:
> 
> 
> 
> 
> Architectural Notes: Always check for the latest designs in Figma

## Testing done

- Content change for vitals should not require a unit test.

## Screenshots

### Side Nav ###

|         | Flag On | Flag Off |
| ------- | ------ | ----- |
| Desktop  |<img width="200" alt="image" src="https://github.com/department-of-veterans-affairs/vets-website/assets/87040148/834cf03e-16cd-4eea-b086-16b0a44418e7">|<img width="200" alt="image" src="https://github.com/department-of-veterans-affairs/vets-website/assets/87040148/54e62750-d034-4506-bba6-4e4ea03f5000">|
| Mobile  |<img width="200" alt="image" src="https://github.com/department-of-veterans-affairs/vets-website/assets/87040148/d5fb9890-5e1e-4a4a-a0b5-d7275fd25c17">|<img width="200" alt="image" src="https://github.com/department-of-veterans-affairs/vets-website/assets/87040148/c1176df9-be94-4e0d-9a2f-39130df893c2">|

### Landing Page Content ###

|         | Flag On | Flag Off |
| ------- | ------ | ----- |
| Desktop  |<img width="200" alt="image" src="https://github.com/department-of-veterans-affairs/vets-website/assets/87040148/da134c53-6929-4184-bac7-baa113467453">|<img width="200" alt="image" src="https://github.com/department-of-veterans-affairs/vets-website/assets/87040148/1e645768-05d9-4c0d-af21-7ce04016586f">|

## What areas of the site does it impact?

MHV Medical Records

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication
